packages/react-data-grid/src/ReactDataGrid.js

```diff
@@ -45,15 +43,15 @@ type RowUpdateEvent = {
   rowIdx: number;
 };
 
+type ColumnMetricsType = {
+  columns: Array<Column>;
+  totalWidth: number;
+  minColumnWidth: number;
+};
+
 const ReactDataGrid = createReactClass({
   displayName: 'ReactDataGrid',
 
-  mixins: [
```

Comment:
``ColumnMetricsMixin mixin is only used by the `ReactDataGrid` component so moved the logic into the component